frontend/src/components/toast/toast-show-hide-input.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import Center from "react-center";
+import ToastButton from "./toast-button";
+import Style from "style-it";
+
+class ToastShowHideInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.content = React.createRef();
+    this.button = React.createRef();
+    this.showInput = this.showInput.bind(this);
+    this.isEmpty = this.isEmpty.bind(this);
+  }
+
+  isEmpty() {
+    let result = true;
+
+    let checkEmpty = React.Children.map(this.props.children, (child, i) => {
+      return !!child.props.value;
+    });
+
+    for (var x of checkEmpty) if (x) result = false;
+
+    return result;
+  }
+
+  showInput() {
+    if (this.content.current.classList.contains("hide")) {
+      this.content.current.classList.remove("hide");
+      this.content.current.classList.add("show");
+      this.button.current.classList.remove("show");
+      this.button.current.classList.add("hide");
+    }
+  }
+
+  render() {
+    const styles = `
+    .hide{
```

Comment:
can you add spaces around the css where needed for better readability and consistency between our css?